.travis.yml

```diff
@@ -1,38 +1,8 @@
----
 language: node_js
 node_js:
-  - "4"
+  - '6'
```

Comment:
No... I just didn't want to figure out build stuff right now to compile it down to work in node 4.